recipes/motionless/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "motionless" %}
+{% set version = "1.3.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/motionless-{{ version }}.tar.gz
+  sha256: 04d029dc2f06bf0301bdc0d1d27347dfaec6efaf2809282a645205059528bf3d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - python >=2.7
+    - setuptools
```

Comment:
It doesn't look like `setuptools` is a run time dependency, so you can remove this here. You do need to add `six` as this is a requirement for the this version.